README.md

```diff
@@ -11,6 +11,140 @@ Inside of the nix-shell:
 - build the documentation by running `make html`
 - view changes as you edit by running `make livehtml` and visit http://127.0.0.1:8000
 
+## Deploying
+
+### Paths
+
+In order for the version switcher to work properly, the paths where we serve the docs have to have the following structure:
+
+- Docs for latest release are at `/`
+  - Ends up being the target for old links, so old links continue to work as they always have
+- Docs for previous releases are at `/vN.N.N/`
+
+For example, if we're publishing v5.4.5, v5.4.4, and v5.4.3...
+
+- `https://docs.gravwell.io` is v5.4.5
+- `https://docs.gravwell.io/v5.4.4/` is v5.4.4
+- `https://docs.gravwell.io/v5.4.3/` is v5.4.3
+
+### Directory structure
+
+Assume we're using a web server to serve a directory as `docs.gravwell.io`.
+
+If we wanted to serve v5.4.5, v5.4.4, and v5.4.3 as shown above, we could structure the directory like this:
+
+```
+docs/     <- the full build of v5.4.5
+├─ 404.html
+├─ index.html
+├─ ...
+├─ v5.4.4   <- the full build of v5.4.4
+│  ├─ 404.html
+│  ├─ index.html
+│  ├─ ...
+├─ v5.4.3   <- the full build of v5.4.3
```

Comment:
should this have `/` after `v5.4.3` to indicate a directory (like `docs/` above)?